packages/integration-tests/projects/suite-web/tests/suite/bug-report-form.test.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+
+
+// @group:suite
+// @retry=2
+
+describe('Stories of bug report forms', function () {
+    beforeEach(function() {
```

Comment:
For consistency with the rest of the code-base it would be better to use arrow function like:


```javascript
    beforeEach(() => {
```